osfoffline/tasks/operations.py

```diff
@@ -80,7 +93,31 @@ def __init__(self, local):
     @asyncio.coroutine
     def run(self):
         print("Delete Local File: {}".format(self.local))
-
+        db_parent = session.query(models.File).filter(models.File.osf_id == self.local.parent.id).one()
+        path = os.path.join(db_parent.path, self.local.name)
+        if not path:
+            raise Exception
+        os.remove(path)
+
+        # get item
+        try:
+            item = _get_item_by_path(path)
```

Comment:
See note below- can re make database deletions a separate task with reusable code? Is that an idea that would make sense for our purposes?